src/test/scala/chiselTests/AutoClonetypeSpec.scala

```diff
@@ -265,19 +265,18 @@ class AutoClonetypeSpec extends ChiselFlatSpec with Utils {
 
   behavior of "Compiler Plugin Autoclonetype"
 
-  // Necessary test for 3.4.x, but we will break this (for non-plugin users) in 3.5
-  it should "NOT break code that extends chisel3.util Bundles (whether they use the plugin or not)" in {
-    class MyModule extends MultiIOModule {
-      val io = IO(new InheritingBundle)
-      io.deq <> io.enq
-      io.count := 0.U
-      io.error := true.B
-    }
-    elaborate(new MyModule)
-  }
-
   // New tests from the plugin
   if (usingPlugin) {
```

Comment:
Do we need this anymore?

Reply:
Yes, the `noPluginTests` compile and run this same file https://github.com/chipsalliance/chisel3/blob/03ec2156988ddef35272365fb86415511eafb4e2/build.sbt#L206

We should consider making the plugin mandatory but that's not the purview of this PR